tools/genmake2

```diff
@@ -1084,6 +1086,134 @@ EOF
     echo " --> set HAVE_NETCDF='$HAVE_NETCDF'" >> $LOGFILE
 }
 
+check_nf90io_mpi_libs()  {
+    if test ! "x$SKIP_NF90IO_CHECK" = x ; then
+	    return
+    fi
+    echo >> $LOGFILE
+    echo "running: check_nf90io_libs()" >> $LOGFILE
+    echo "First looking for hdf5 in nc-config:" >> $LOGFILE
+    echo "running: $NETCDF/bin/nc-config --has-hdf5" >> $LOGFILE
+    if [[ $($NETCDF/bin/nc-config --has-hdf5) != "yes" ]]; then
+      echo "nc-config could not fine hdf5" >> $LOGFILE
+      return
+    fi
+    echo "         ...returns yes" >> $LOGFILE
+
+    echo "Looking for h5pcc:" >> $LOGFILE
+    echo "running: $HDF5/bin/h5pcc -showconfig | grep Parallel" >> $LOGFILE
+    if [[ $($HDF5/bin/h5pcc -showconfig | grep Parallel) != *yes* ]]; then
+      echo "Somehow h5pcc not compiled w/ parallel" >> $LOGFILE
+      return
+    fi
+    echo "         ...returns yes" >> $LOGFILE
```

Comment:
Do we really need these new tests?  I would just test compilation of f90tst_parallel.f90 and maybe on failure print some hints to help the user figure out whether she has parallel netcdf support.  netcdf 4.5, for instance, has `nc-config --has-parallel`.  I don't like the use of `$NETCDF` and `$HDF5` too much as most optfiles do not define or use these right now.

Reply:
Ah, I remember now.  The issue is that these tests *don't* fail if parallel support is not enabled.  And if I remember correctly the error was a bit mysterious to the end user.  

OTOH, I agree that there doesn't seem to be an easily robust way to check for parallel netcdf.  You can always call `nc-config` but it is hard to tell if it is the right `nc-config` on a supercomputer with many possible modules.  Thats why I was suggesting the environment variables.  Some supercomputers seem to set these environment variables, and if they don't I don't think its a heavy ask to ask the user to set them, rather than just providing bare `$LIB` and `$INC` lists.  But, I don't think sorting all that out should hold this up, just something to consider.